test/conftest.py

```diff
@@ -31,6 +33,8 @@ def pytest_addoption(parser):
                      help='CQL server port to connect to')
     parser.addoption('--ssl', action='store_true',
                      help='Connect to CQL via an encrypted TLSv1.2 connection')
+    parser.addoption('--dc_rf', action='store', default='1',
+                     help='Default datacenter replication factor')
```

Comment:
Ok. I'm not thrilled with this approach but can accept it. I would have liked to think that we can figure out how many nodes exist in the cluster from within the test, instead of it being passed from the test.py configuration. Making this a configuration option can allow users to (accidentally) create non-sensical setups, like just one node and RF=3.

Reply:
removed in v7